carina-api/src/main/java/com/qaprosoft/apitools/builder/MessageBuilder.java

```diff
@@ -23,9 +23,12 @@
 import freemarker.template.Configuration;
 import freemarker.template.Template;
 import freemarker.template.TemplateException;
+import org.apache.log4j.Logger;
 
 public class MessageBuilder {
 
+    private static final Logger LOGGER = Logger.getLogger(MessageBuilder.class);
```

Comment:
still log4j directly? that's valid declaration of logger:
` private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());`